bia_export/cli.py

```diff
@@ -172,7 +173,12 @@ def study_uuid_to_export_ai_dataset(study_uuid: str) -> ExportAIDataset:
 
     bia_study = rw_client.get_study(study_uuid, apply_annotations=True)
 
-    images = get_images_with_a_rep_type(study_uuid, "ome_ngff")
+    # Get OME-NGFF images only
+    images = get_images_with_a_rep_type(study_uuid, "ome_ngff",limit=10)
```

Comment:
Do you have a ticket to link to for this change? Just because i see a limit here and so i want to ask if it's correct to have the limit here without context for what this change is aiming to do.

Reply:
The limit was always there (in the get_images_with_a_rep_type function) but was not overridden explicitly. I set 10 because we usually convert more images for datasets in specific galleries like AI Gallery. It will change in the future to a bigger number, when we finalise the design decision of converting all images for datasets in specific galleries and implement that.